platform/windows/os_windows.cpp

```diff
@@ -2926,7 +2940,8 @@ bool OS_Windows::is_disable_crash_handler() const {
 Error OS_Windows::move_to_trash(const String &p_path) {
 	SHFILEOPSTRUCTW sf;
 	WCHAR *from = new WCHAR[p_path.length() + 2];
-	wcscpy(from, p_path.c_str());
+	wcscpy_s(from, (p_path.length() + 2) / sizeof(WCHAR), p_path.c_str());
```

Comment:
I think I messed this up. It should just be `p_path.length() + 1`, since that's the number of elements.